contracts/src/v0.8/ccip/offRamp/OffRamp.sol

```diff
@@ -604,9 +604,12 @@ contract OffRamp is ITypeAndVersion, MultiOCR3Base {
     // 3. If the receiver is a contract but it does not support the IAny2EVMMessageReceiver interface.
     //
     // The ordering of these checks is important, as the first check is the cheapest to execute.
+    //
+    // To prevent message delivery bypass issues, a modified version of the ERC165Checker is used exclusively here
```

Comment:
Comment claims exclusive usage but it's also used in the token pool call. Probably not relevant to mention if something is exclusively used here